src/modules/queue/components/Queue.jsx

```diff
@@ -6,101 +6,147 @@ import YoutubeResult from '../../uniplayer/Search/YoutubeResult';
 import queue_img from './images/queue.svg';
 import x_img from './images/x.svg';
 
-export default function Queue({isQueueOpen, isSearchOpen, queue, toggleQueue, toggleSearch}) {
-	if (isQueueOpen) {
-		return (
-			<Sidebar expand className="sidebar">
-				<h2>Tracklist</h2>
-				<Search />
-				<QueueList>{queue.map(track => <YoutubeResult result={track.content} key={track.id} highRes />)}</QueueList>
-				<QueueButton onClick={toggleQueue} open>
-					<img src={x_img} />
-				</QueueButton>
-			</Sidebar>
-		);
-	}
+export default function Queue({isSearchOpen, queue, toggleSearch}) {
 	return (
-		<Sidebar className="sidebar">
-			<QueueButton onClick={toggleQueue}>
-				<OButton />
-			</QueueButton>
-		</Sidebar>
+		<QueueOuter>
+			<QueueInner expand className="queue">
+				<QueueHeader>
+					<CollectionRecord>
+						<span />
+					</CollectionRecord>
+					<CollectionInfo>
+						<h5>COLLECTION</h5>
+						<h1>/example</h1>
+						<AddSong onClick={toggleSearch}>
+							Add Song
+							<span> +</span>
+						</AddSong>
+					</CollectionInfo>
+					{!isSearchOpen && <h4>{queue.length} tracks</h4>}
+					{isSearchOpen && <h4>Search for a song on YouTube or Soundcloud</h4>}
+				</QueueHeader>
+				<QueueList>{queue.map(track => <YoutubeResult result={track.content} key={track.id} highRes />)}</QueueList>
+				{isSearchOpen && <Search />}
+			</QueueInner>
+		</QueueOuter>
 	);
 }
 
-const Sidebar = styled.div`
-	background: rgba(29, 29, 29, 0.995);
-	position: absolute;
-	height: 100%;
-	transition: all 0.1s;
-	right: 0;
-	width: ${({expand}) => (expand ? '25rem' : '0rem')};
+const QueueOuter = styled.div`
+	background: rgb(25, 25, 25);
+	width: 100%;
+`;
 
-	h2 {
-		display block;
-		color: #f2f2f2;
-		margin: 1.75rem 0.875rem;
-	}
+const QueueInner = styled.div`
+	display: block;
+	left: 50%;
+	max-width: 75rem;
+	padding-top: 4rem;
+	position: relative;
+
+	transform: translateX(-50%);
+	width: 80%;
 `;
 
 const QueueList = styled.div`
-	max-height: calc(100% - 8.75rem);
-	position: relative;
+	max-height: calc(100vh - 18.5rem);
```

Comment:
I really hate this constantly adjust the exact size of the offset. We either need to give these guys fixed heights or make them work with variables.